meson.build

```diff
@@ -172,6 +172,13 @@ if get_option('enable-cublas')
   extra_defines += '-DUSE_CUBLAS=1'
 endif
 
+if get_option('platform') != 'tizen'
```

Comment:
I don't think we always use encoder. It only depends on the application Pico. So please remove the running shell script here and add it to the applications.

Reply:
Thank you for Review - Fix